web/deploy/terraform/staging/main.tf

```diff
@@ -13,16 +13,16 @@ terraform {
 
 
 module "shared_resources" {
-  source      = "../modules/shared_resources"
+  source = "../modules/shared_resources"
```

Comment:
We won't be calling this module from each environment because otherwise we'll end up with duplicate resources. You'll want to extract this call to its own file that we run only once